resources/lib/ps_vue.py

```diff
@@ -180,8 +180,10 @@ def list_show(show):
 
     if str(show['airings'][0]['badge']) == 'live':
         add_stream(name, channel_url, icon, fanart, info, properties, show_info)
+        add_sort_methods(int(sys.argv[1])
```

Comment:
Can you switch this to use the addon_handle global variable instead?

Reply:
Fixed... sorry about that, I assumed that addon_handle and int(sys.argv[1]) were different because the add_stream method referenced int(sys.argv[1])... I like the whole thing better now that just addon_handle can be used :)

Reply:
no problem. I didn't realize add_stream referenced that way. I'll need to fix that. Thanks.